spec/IS20.md

```diff
@@ -97,19 +127,19 @@ type TransactionStatus = variant {
 };
 
 type TxRecord = record {
-  to : principal;
-  fee : nat;
+  to : AccountIdentifier;
+  fee : Tokens128;
   status : TransactionStatus;
-  from : principal;
+  from : AccountIdentifier;
```

Comment:
lets add:
from_details:  opt (Principal, subaccount)

Reply:
to_details:  opt (Principal, subaccount)

Reply:
Ok on it